content/release.rst

```diff
@@ -37,6 +37,21 @@ Such a release can have a normal version... and if changes are made
 during the Q&A review a new patch version is released.
 
 
+Release Planning
+================
+
+Tickets marked for the 'next-release' milestone are technically to be
+covered for the next release but in most situations these will fall beyond
+the next release.
+Tickets that must be in the 'next-release' is set with the high priority tag.
```

Comment:
is set with priority `High`. 

to make it clear that this is the exact value of the priority.